GroceryStore/ProductDetailViewController.m

```diff
@@ -0,0 +1,123 @@
+//
+//  ProductDetailViewController.m
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "ProductDetailViewController.h"
+
+@interface ProductDetailViewController ()
+
+@property (weak, nonatomic) IBOutlet UILabel *productTitleLabel;
+@property (weak, nonatomic) IBOutlet UILabel *productQuantityLabel;
+
+@property (weak, nonatomic) IBOutlet UILabel *quantityAdjustmentLabel;
+@property (weak, nonatomic) IBOutlet UIStepper *stepperOutlet;
+
+@property (strong, nonatomic) NSMutableURLRequest *URLRequest;
+@property (strong, nonatomic) NSString *URLString;
+@property (strong, nonatomic) NSURLSession *session;
+
+@end
+@implementation ProductDetailViewController
+
+- (void)viewDidLoad {
+    
+    self.productTitleLabel.text = self.productName;
+    self.productQuantityLabel.text = [NSString stringWithFormat:@"Current quantity is: %@", self.quantity];
+}
+
+- (IBAction)stepped:(UIStepper *)sender {
+    
+    self.quantityAdjustmentLabel.text = [NSString stringWithFormat:@"%.0f", sender.value];
+    
+}
+
+- (IBAction)removeAllStock:(id)sender {
+    
+    //create session
+    [self createSessionWithURLComponent:@"inventory"];
+    [self createURLRequestWithHTTPMethod:@"DELETE"];
+    
+    //create task
+    NSURLSessionDataTask *deleteProduct = [self.session dataTaskWithRequest:self.URLRequest completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+        
+        [self createAndHandleURLResponse:response withData:data];
+    }];
+    
+    //resume
+    [deleteProduct resume];
+}
+
+- (IBAction)restockProduct:(id)sender { //add
+    //TODO? Can't stock product not currently/previously in inventory (201)
```

Comment:
how could you change `createAndHandleURLResponse` to allow this to work?
hint: http://en.wikipedia.org/wiki/List_of_HTTP_status_codes